docs/zh/README.md

```diff
@@ -65,15 +65,15 @@ Java客户端不依赖任何框架，能够运行于所有Java运行时环境，
   * Apollo还提供了打包脚本，一键就可以生成所有需要的安装包，并且支持自定义运行时参数
 
 # Usage
-  1. [应用接入指南](zh/usage/apollo-user-guide)
-  2. [Java客户端使用指南](zh/usage/java-sdk-user-guide)
-  3. [.Net客户端使用指南](zh/usage/dotnet-sdk-user-guide)
-  4. [Go、Python、NodeJS、PHP等客户端使用指南](zh/usage/third-party-sdks-user-guide)
-  5. [其它语言客户端接入指南](zh/usage/other-language-client-user-guide)
-  6. [Apollo开放平台接入指南](zh/usage/apollo-open-api-platform)
+  1. [应用接入指南](zh/portal/apollo-user-guide)
+  2. [Java客户端使用指南](zh/client/java-sdk-user-guide)
+  3. [.Net客户端使用指南](zh/client/java-sdk-user-guide)
+  4. [Go、Python、NodeJS、PHP等客户端使用指南](zh/client/third-party-sdks-user-guide)
```

Comment:
The `/zh/client/third-party-sdks-user-guide` was removed.

![image](https://github.com/apolloconfig/apollo/assets/837658/735b6855-c274-4104-8444-2b496a05b950)

Reply:
I think we should remove this part from `README.md` because the catalog is just on the left side. We don't need to maintain this catalog mantually.